File: ServerWebJava/src/main/java/com/example/serverwebjava/models/Account.java
package com.example.serverwebjava.models;

import java.io.Serializable;

public class Account implements Serializable {
    private Integer id;
    private String email;
    private String password;
    private String role;
    private String firstname;
    private String lastname;

    public Account() {}
    public Account(Integer id, String email, String password, String role, String firstname, String lastname) {
        this.id = id;
        this.email = email;
        this.password = password;
        this.role = role;
        this.firstname = firstname;
        this.lastname = lastname;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }

    public String getFirstname() {
        return firstname;
    }

    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }
}

File: UACMicroservice/src/main/kotlin/com/example/uacmicroservice/models/Account.kt
package com.example.uacmicroservice.models

import javax.persistence.*

@Entity
@Table(name = "Accounts")
class Account(id: Int, email: String, password: String, role: String, firstname: String, lastname: String) {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private var id: Int = 0
    private var email: String = ""
    private var password: String = ""
    private var role: String = ""
    private var firstname: String = ""
    private var lastname: String = ""

    init {
        this.id = id
        this.email = email
        this.password = password
        this.role = role
        this.firstname = firstname
        this.lastname = lastname
    }

    fun getId(): Int{
        return id
    }

    fun getEmail(): String {
        return email
    }

    fun getPassword(): String {
        return password
    }

    fun getRole(): String {
        return role
    }

    fun getFirstName(): String {
        return firstname
    }

    fun getLastName(): String {
        return lastname
    }

    fun setId(id: Int){
       this.id = id
    }

    fun setEmail(email: String){
        this.email = email
    }

    fun setPassword(password : String){
        this.password = password
    }

    fun setRole(role: String){
        this.role = role
    }

    fun setFirstName(firstname: String){
        this.firstname = firstname
    }

    fun setLastName(lastname : String){
        this.lastname = lastname
    }
}